asmcli/commands/create-mesh.sh

```diff
@@ -1,11 +1,17 @@
 create-mesh_subcommand() {
   ### Preparation ###
+
+  # using kubeconfig globally for create-mesh sub-command
+  context_set-option "KUBECONFIG_SUPPLIED" 1
```

Comment:
I'd like to either change the name of this context variable, or change the logic that depends on it.

Reply:
why/to what?